src/main/kotlin/org/neo4j/graphql/Translator.kt

```diff
@@ -18,11 +21,56 @@ class Translator(val schema: GraphQLSchema) {
                                                  val mutation: CRUDConfig = CRUDConfig())
     data class CRUDConfig(val enabled:Boolean = true, val exclude: List<String> = emptyList())
     data class Cypher( val query: String, val params : Map<String,Any?> = emptyMap()) {
+        fun with(p: Map<String,Any?>) = this.copy(params = this.params + p)
+        fun escapedQuery() = query.replace("\"","\\\"").replace("'","\\'")
         companion object {
             val EMPTY = Cypher("")
+
+            private fun createRelationshipQuery(keyword: String, sourceVariable: String, sourceLabel: String, sourceId: String,
+                                           isList: Boolean, targetFieldName: String, targetVariable: String,
+                                           targetLabel: String, targetId: String, relationshipDirection: String,
+                                           relationshipType: String): String {
+                return """
+                    MATCH ($sourceVariable:${sourceLabel.quote()} {$sourceId:$$sourceId})${
```

Comment:
I would probably split it into two statements for the isList and not

and then just append the `$keyword ...` bit